cmd/skopeo/inspect.go

```diff
@@ -85,9 +96,25 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 		return err
 	}
 
-	src, err := parseImageSource(ctx, opts.image, imageName)
-	if err != nil {
-		return fmt.Errorf("Error parsing image name %q: %v", imageName, err)
+	if _, err := alltransports.ParseImageName(imageName); err != nil {
+		return err
+	}
+
+	if err := retryIfNecessary(ctx, func() error {
+		source, err := parseImageSource(ctx, opts.image, imageName)
+		src = source
+		if err != nil {
```

Comment:
(Absolutely non-blocking, just wondering and asking for opinions: Would it be more readable to have
```go
src, err = parseImageSource…
```
?

That’s one line fewer on every call, OTOH there is a risk that we’ll habitually write `:=` and break things.)

WDYallThink?